src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -272,6 +275,12 @@ private static (string severityLabel, NuGetLogCode code) GetSeverityLabelAndCode
                                 continue;
                             }
 
+                            if (SuppressedAdvisories?.Contains(knownVulnerability.Url.OriginalString) == true)
+                            {
+                                SuppressedAdvisoriesUsedCount++;
```

Comment:
Do you expect to compare used vs defined at all?


Because the way this is calculated right now, it doesn't handle multi targeting, or scenarios where 1 advisory refers to multiple packages etc. 

I think it'd be  helpful defining what you want this number to be

Reply:
Instead of the single 'Used' metric, we're now recording two separate metrics here: the total number of warnings we suppressed during audit: `"Audit.SuppressedAdvisories.TotalWarningsSuppressed.Count"`, and the number of distinct advisories we suppressed: `"Audit.SuppressedAdvisories.DistinctAdvisoriesSuppressed.Count"`